src/com/facebook/buck/features/go/GoTest.java

```diff
@@ -159,9 +162,14 @@ public GoTest(
 
   private ImmutableList<TestResultSummary> parseTestResults() throws IOException {
     ImmutableList.Builder<TestResultSummary> summariesBuilder = ImmutableList.builder();
+    CharsetDecoder decoder = Charsets.UTF_8.newDecoder();
+    decoder.onMalformedInput(CodingErrorAction.REPLACE);
+    decoder.replaceWith("囧");
```

Comment:
whats with this character?

Reply:
Something to use as the placeholder for non-printable characters. If you don't like this one, please suggest another one, I will replace.

Reply:
lets make this atleast a named constant